core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -278,6 +278,14 @@ typedef enum WASMOpcode {
     DEBUG_OP_BREAK = 0xdc, /* debug break point */
 #endif
 
+#if (WASM_ENABLE_JIT != 0                                         \
+     || (WASM_ENABLE_FAST_INTERP != 0 && WASM_ENABLE_SIMDE != 0)) \
+    && WASM_ENABLE_SIMD != 0
```

Comment:
Not sure whether it requires WASM_ENABLE_JIT and WASM_ENABLE_SIMD? Can we change to `#if WASM_ENABLE_FAST_INTERP != 0 && WASM_ENABLE_SIMDE != 0`?